chain/params.go

```diff
@@ -90,6 +90,7 @@ const (
 	QuorumCalcAlignment = "quorumcalcalignment"
 	TxHashWithType      = "txHashWithType"
 	EIP2929             = "EIP2929"
+	EIP2930             = "EIP2930"
 	EIP2565             = "EIP2565"
```

Comment:
I think we should merge these forks into single, "berlin" fork. WDYT?

Reply:
yeah I will do that after implementing all EIPs related to Berlin fork